src/insns/cbo_exceptions.adoc

```diff
@@ -19,13 +24,12 @@ endif::[]
 
 ifdef::cbo_clean_flush[]
 | Permission violation  | It does not grant <<w_perm>> and <<r_perm>>
-| Length violation      | None of the bytes accessed are within the bounds
 endif::cbo_clean_flush[]
 
 ifdef::cbo_inval[]
 | Permission violation  | It does not grant <<w_perm>>, <<r_perm>> or <<asr_perm>>
-| Length violation      | At least one byte accessed is outside the bounds
 endif::[]
+| Length violation      | None of the bytes accessed are within the bounds
```

Comment:
I think inval should still have the same check as zero, i.e. full cache line. Personally, I'd like it to require almighty but since I can make it behave as flush if I control the software stack that may not be necessary.

Reply:
Agreed, it would be best to require that all bytes are within bounds for CBO.INVAL just as we do for CBO.ZERO.

Reply:
This was not resolved?